.github/workflows/onnx.yml

```diff
@@ -11,10 +11,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
 
     - name: install .NET
-      uses: actions/setup-dotnet@v2
+      id: install-dotnet
+      uses: actions/setup-dotnet@v4
```

Comment:
Can you look for these actions in all the `.github/workflows` files? Update the entire repo as needed.